try-slang.js

```diff
@@ -217,11 +249,96 @@ async function render(timeMS)
         requestAnimationFrame(render);
 }
 
+
+
+// This is the definition of the printf buffer.
+// struct FormatedStruct
+// {
+//     uint32_t type = 0xFFFFFFFF;
+//     uint32_t low = 0;
+//     uint32_t high = 0;
+// };
+//
+function parsePrintfBuffer()
```

Comment:
Can we move this function to util.js?

Reply:
This function is previously a member of ComputePipeline, I don't have problem that you moved it out from that class, but it's actually a utility function. try-slang.js should only contain the main control logic.

Reply:
Moved this to util